### ISSUE TITLE
Support Darwin arm64 platform

### ISSUE THREAD

**Issue by shutangyu** [akash-akya/ex_cmd#15]

**Comment by akash-akya** [comment]
Hi @shutangyu, the concern is pre-built binary right? 
It should work without binary as well, but it requires golang tooling to be present in the host machine so that it can compile the binary.

**Comment by shutangyu** [comment]
Thx for ur fast reply

> the concern is pre-built binary right?

Yes.

> It should work without binary as well, but it requires golang tooling to be present in the host machine so that it can compile the binary.

I see, but it make our development machine's setup complicated. so IMO is better to support it

**Comment by akash-akya** [comment]
Yes, agree on supporting. Just wanted to confirm the issue.

Will add binary and push changes soon

**Comment by shutangyu** [comment]
Thank you so much! we used `ex_cmd` in production and made our code elegant and effective. It would be a great honor if I can buy you a coffee.

**Comment by akash-akya** [comment]
Glad to know that you are finding it useful!

> It would be a great honor if I can buy you a coffee

Thanks for the thought! It means a lot :)

Regarding the binary, I added it [here](https://github.com/akash-akya/ex_cmd/pull/16). But I don't have M1 machine to test. It would be great if you can verify it once. I'll merge and cut release once you confirm. branch: `mac-arm64-bin`

Please make sure you **_don't_** have `go` binary in the path. By default, ex_cmd compiles the binary whenever it can, instead of using pre-built binary.

**Comment by akash-akya** [comment]
@shutangyu  bumping the thread, just in case if you missed 😅 

Just checking once with this should suffice

```elixir
{:ex_cmd, git: "https://github.com/akash-akya/ex_cmd.git", branch: "mac-arm64-bin"}
```

**Comment by shutangyu** [comment]
@akash-akya Thank you! We tested in our M1 machine and works well.

**Comment by akash-akya** [comment]
@shutangyu  thanks for confirming. Published v0.7.0 with pre-built M1 binaries
